rdiodj/static/css/style.less

```diff
@@ -573,6 +525,7 @@ img {
 
 .presence {
 	margin-left: -15px;
+	margin-bottom: 0.4em;
```

Comment:
I try to do things in ems as much as possible. Since this margin deals with spacing for text, I thought ems were more appropriate.